ui/cryptomaterial/segmented_control.go

```diff
@@ -42,7 +42,7 @@ func (sc *SegmentedControl) Layout(gtx C) D {
 	return LinearLayout{
 		Width:      WrapContent,
 		Height:     WrapContent,
-		Background: sc.theme.Color.Background,
+		Background: sc.theme.Color.Gray2,
```

Comment:
What's the motivation for this change?

Reply:
Figma design.

Reply:
<img width="852" alt="image" src="https://github.com/crypto-power/cryptopower/assets/27733432/49ce4cd3-4d38-450a-a302-2f8ffc33ec61">
that is gray 2.